src/annotator/util/buckets.js

```diff
@@ -144,11 +145,10 @@ export function constructPositionPoints(anchors) {
  * highlights and group them into a collection of "buckets".
  *
  * @param {PositionPoint[]} points
- * @return {BucketInfo}
+ * @return {Bucket[]}
```

Comment:
There is no change to the algorithm in this module for building buckets, just the shape of the object(s) it's operating on.